pkg/scheduler/api/latest/latest.go

```diff
@@ -1,56 +0,0 @@
-/*
```

Comment:
This file doesn't need to exist as we use a unified `scheme.Scheme.Codec` to deal with encoding/decoding logic.